meteor/client/lib/shelf.ts

```diff
@@ -13,6 +13,12 @@ import { ScanInfoForPackages } from '../../lib/mediaObjects'
 import { processAndPrunePieceInstanceTimings } from '@sofie-automation/corelib/dist/playout/infinites'
 import { getUnfinishedPieceInstancesReactive } from './rundownLayouts'
 
+export interface ShelfDisplayOptions {
+	buckets: boolean
+	layout: boolean
+	inspector: boolean
```

Comment:
Since all of these attributes are booleans, it would be nice if their name reflected that: `hasBuckets`, `hasLayout` and `hasInspector`  (or use "is" if that's the appropiate prefix)

If I see:
``` 
const buckets = shelfDiplayOptions.buckets;
```
I would expect that my variable "buckets" would be a list of something, not a boolean

Reply:
Only `enable` works kind of okay semantically in this case

Reply:
`enable` works, but since it is a verb I would expect it to be function that does something.
But it's way better than not having a prefix at all :)